Makefile

```diff
@@ -17,7 +17,7 @@ OUTPUT_URL = https://docker-dev03.pcic.uvic.ca/wpsoutputs
 SANITIZE_FILE := https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/raw/master/notebooks/output-sanitize.cfg
 
 .PHONY: all
-all: develop test clean-test test-notebooks
+all: develop test clean-test test-notebooks-online
```

Comment:
How can I run the old `test-notebooks` separately? I think it would be helpful if there is an option to run the test.

Reply:
When you say old I assume you mean what we were using before? `make test-notebooks-online` will target the docker instance whereas `make test-notebooks` will target a local instance.